.github/workflows/pr.yaml

```diff
@@ -127,8 +127,8 @@ jobs:
       - name: Verify the image signing
         if: ${{ github.event_name != 'pull_request' && github.repository == 'weaveworks/weave-gitops' }}
         run: |
```

Comment:
Change to > and remove indents instead, to avoid the backslashes?

Reply:
Fixed

Reply:
Think you have to remove indents as well, that's the downside of >

Reply:
Then I actually prefer the backslash solution.